### PR TITLE
franklin-server endpoint

### PR DESCRIPTION
**Pull request by oudeismetis**
Connected to #89 

As this is a new endpoint, I assume it is fine to add it to `v1`.

`GET /v1/domains/?domain=example.com`

### CODE REVIEW

**Comment by oudeismetis** [comment]
image

**Comment by oudeismetis** [comment]
Response will be blank if the domain exists, but nothing is deployed to it.
Error will be returned for all other cases.

**Comment by JulianGindi** [comment]
👍

File: franklin/builder/views.py [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
 
 
+@api_view(['GET'])
+@permission_classes((AllowAny, ))
+def domain(request):
+    """
+    Returns details about a domain managed by Franklin
+    """
+    if request.method == 'GET':
+        domain = request.GET.get('domain', False)
```
Comment (jcarbaugh) [inline franklin/builder/views.py]: You can remove the default of `False`. `get()` returns `None` if the key is not found so you can just rely on that default behavior.

File: franklin/builder/views.py [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+        if domain:
+            environment = get_object_or_404(Environment, url=domain)
+            serializer = BuildSerializer(environment.get_current_deploy())
```
Comment (jcarbaugh) [inline franklin/builder/views.py]: What will this serialize if `get_current_deploy() is None`? An empty JSON object or will it throw an error?

Comment (oudeismetis) [inline franklin/builder/views.py]: Related to my comment above, this is where it'll just be an empty string if it doesn't have a current_deploy.

File: franklin/builder/views.py (hunk 1 above)
Comment (oudeismetis) [inline franklin/builder/views.py]: will do

**Comment by jcarbaugh** [comment]
👍

**Pull request merged**